discovery-provider/ddl/migrations/0010_usdc_userbank_tables.sql

```diff
@@ -0,0 +1,62 @@
+BEGIN;
+
+-- USDC user bank accounts
+CREATE TABLE IF NOT EXISTS usdc_user_bank_accounts (
+    "signature" varchar NOT NULL,
+    ethereum_address varchar NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    bank_account varchar NOT NULL,
+    PRIMARY KEY("signature")
+);
+CREATE INDEX IF NOT EXISTS idx_usdc_user_bank_accounts_eth_address ON usdc_user_bank_accounts USING btree (ethereum_address);
+
+-- USDC transactions history
+CREATE TABLE IF NOT EXISTS usdc_transactions_history (
+    user_bank varchar NOT NULL,
+    slot integer NOT NULL,
+    "signature" varchar NOT NULL,
+    transaction_type varchar NOT NULL,
+    method varchar NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    transaction_created_at timestamp NOT NULL,
+    change numeric NOT NULL,
+    balance numeric NOT NULL,
+    tx_metadata varchar,
+    PRIMARY KEY(user_bank, "signature")
+);
+CREATE INDEX IF NOT EXISTS idx_usdc_transactions_history_slot ON usdc_transactions_history USING btree (slot);
+CREATE INDEX IF NOT EXISTS idx_usdc_transactions_history_type ON usdc_transactions_history USING btree (transaction_type);
+
+-- USDC purchases table
+DO $$ BEGIN
+    CREATE TYPE usdc_purchase_content_type AS ENUM ('track', 'playlist', 'album');
+EXCEPTION
+    WHEN duplicate_object THEN null;
+END $$;
+
+CREATE TABLE IF NOT EXISTS usdc_purchases (
+    slot integer NOT NULL,
+    "signature" varchar NOT NULL,
+    buyer_user_id integer NOT NULL,
+    seller_user_id integer NOT NULL,
+    amount numeric NOT NULL,
```

Comment:
this was bigint in the design doc. I could make it bigint here if wanted? I'm guessing it came from tips which uses bigint. existing audio_transaction_history uses numeric. What do y'all think? Numeric or bigint?

Reply:
keeping it as bigint

Reply:
Agreed. It's a wei value, so I don't think we want/need floating point precision here.
Plus bonus, users can transfer 9 trillion usdc if they want!

Reply:
Do we also then want the fields in the transactions history to be bigints?